elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/DisableAggStoreAsyncTest.java

```diff
@@ -5,12 +5,115 @@
  */
 package example.tests;
 
-import org.springframework.test.context.ActiveProfiles;
+import static com.yahoo.elide.Elide.JSONAPI_CONTENT_TYPE;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attr;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attributes;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.data;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.id;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.resource;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.type;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.yahoo.elide.core.exceptions.HttpStatus;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlMergeMode;
+
+import io.restassured.response.Response;
+
+import javax.ws.rs.core.MediaType;
 
 /**
  * Executes Async tests with Aggregation Store disabled.
  */
-@ActiveProfiles("disableAggStore")
-public class DisableAggStoreAsyncTest extends AsyncTest {
+@SqlMergeMode(SqlMergeMode.MergeMode.MERGE)
+@Sql(
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+        scripts = "classpath:db/test_init.sql",
+        statements = "INSERT INTO ArtifactGroup (name, commonName, description, deprecated) VALUES\n"
+            + "\t\t('com.example.repository','Example Repository','The code for this project', false);"
+)
+@TestPropertySource(
+        properties = {
+                "elide.aggregation-store.enabled=false"
+        }
+)
+public class DisableAggStoreAsyncTest extends IntegrationTest {
+
+    @Test
+    public void testAsyncApiEndpoint() throws InterruptedException {
```

Comment:
What is the goal of this test exactly?  Verify that async works when agg store is disabled?  We should document that.

Reply:
Yes, will update.